### ISSUE TITLE
Create ~/.gitconfig if missing

### ISSUE THREAD

**Issue by dougireton** [Nordstrom/chefdk_bootstrap#51]
Use this template to create a ~/.gitconfig for the user. We will need to prompt the user for name and email.

The git cookbook has a `git_config` LWRP we should look at.

```
[user]
        name = Jane Doe
        email = janedoe@example.com
[core]
        editor = atom --wait
        autocrlf = True

[alias]
        co = checkout
        br = branch
        ci = commit
        st = status
        lol = log --color --graph --pretty=format:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr) %C(bold blue)<%an>%Creset' --abbrev-commit
```

**Comment by dougireton** [comment]
Also set `push.default`:

```
warning: push.default is unset; its implicit value has changed in
Git 2.0 from 'matching' to 'simple'.

To squelch this message and adopt the new behavior now, use:

  git config --global push.default simple
```

**Comment by dougireton** [comment]
On Mac use this:

```
git config --global credential.helper osxkeychain
```

**Comment by MarkGibbons** [comment]
Fixed by #157
